src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -1,10 +1,18 @@
 {% trans_default_domain "locationview" %}
 
 {% if draftVersions %}
+    {{ form_start(draftActionsForm, {"action": path("ez_hybrid_platform_ui_content_view_draft_actions"), 'attr': {'class': 'ez-js-standard-form'}}) }}
+
     <div class="ez-list-toolbar">
         <h2 class="ez-list-toolbar-label">{{ 'locationview.versions.draft.under.edit'|trans()|desc('Draft under edit') }}</h2>
+        <div class="container-buttons">
```

Comment:
Ignore my previous comment 😄 I think the issue was my browser window was too small and therefore it was stacking them on top of each other.